tutorials/suspend-rendering.html

```diff
@@ -0,0 +1,351 @@
+      To suspend the rendering process you can use <code>suspendExecution</code> method just before the actions you want to batch.
+      This is a manual approach - after suspending you have to remember to resume the process with the <code>resumeExecution</code> method.
+      <pre>
+        <code class="lang-js">
+hot.suspendExecution();
+const filters = hot.getPlugin('filters');
+
+filters.addCondition(2, 'contains', ['3']);
+filters.filter();
+hot.getPlugin('columnSorting').sort({ column: 1, sortOrder: 'desc' });
+hot.resumeExecution(); // It updates the cache internally
+        </code>
+      </pre>
+
+  <div class="example-container clearfix" name="suspend-demo">
+    <h3 id="page-suspend-demo">Live demo of suspend feature</h3>
+    <p>The following examples show how much the <code>batch</code> method can decrease the render time.
+      Both of the examples share the same dataset and operations. The first one shows how much time lapsed
+      when the <code>batch</code> method was used. You can run the second example to check how much it takes
+      to render without the <code>batch</code> method.
+
+    </p>
+
+      <div data-jsfiddle="example1">
+        <div id="example1"></div>
+        <button id="buttonWithout" onclick="standardAlterTable">Without batch</button>
+        <button id="buttonWith" onclick="batchedAlterTable">With batch</button>
+        <div id="logOuput"></div>
+      </div>
+
+      <div class="codeLayout">
+        <div class="buttons">
+          <button class="jsFiddleLink" data-runfiddle="example1">
+            <i class="fa fa-jsfiddle"></i>
+            Edit
+          </button>
+          <button class="dump" name="dump" data-dump="#example1" data-instance="hot1"
+                  title="Print current data source to console">
+            <i class="fa fa-terminal"></i>
+            Log to console
+          </button>
+        </div>
+        <style data-jsfiddle="example1">
+            .handsontable .green-bg {
+              background-color: #2ECC40;
+            }
+
+            .handsontable .red-bg {
+              background-color: #FF4136;
+            }
+
+            #buttonWith {
+              background-color: #104ACC;
+              color: #F5F5F5;
+              border: #E0E0E0;
+              padding: 10px;
+              border-radius: 3px;
+              margin-top: 10px;
+            }
+
+            #buttonWithout {
+              background-color: #568afb;
+              color: #F5F5F5;
+              border: #487ff5;
+              padding: 10px;
+              border-radius: 3px;
+              margin-top: 10px;
+            }
+
+            #buttonWith:hover {
+              background-color: #0039B5;
+              cursor: pointer;
+            }
+
+            #buttonWithout:hover {
+              background-color: #3e78f7;
+              cursor: pointer;
+            }
+
+            #logOuput {
+              padding: 15px 0 10px 0;
+              max-height:400px;
```

Comment:
```suggestion
              max-height:400px;
              overflow-y: auto;
```

<img width="660" alt="Screenshot 2021-01-26 at 14 45 51" src="https://user-images.githubusercontent.com/4718998/105852848-4255c980-5fe5-11eb-863c-f5388d28168b.png">